### PR TITLE
Canada adjustments

### PR DESCRIPTION
**Pull request by AnonymousRevan**

### CODE REVIEW

File: common/national_focus/canada.txt [hunk 1]
```diff
@@ -2576,36 +2488,40 @@ focus_tree = {
...
+			332 = {
+				add_extra_state_shared_building_slots = 1
+				add_building_construction = {
+					type = industrial_complex
+					level = 1
+					instant_build = yes
 				}
-				custom_effect_tooltip = lar_decryption_bonus_tt
 			}
+			FRA = {
```
Comment (Mookubooku) [inline common/national_focus/canada.txt:2508]: Why are we buffing France in the Canada tree?

**Comment by GrenadierSchubes** [comment]
Good question, should we have one buff USA and one Buff UK instead?  Would make more sense

**Comment by AnonymousRevan** [comment]
> Good question, should we have one buff USA and one Buff UK instead? Would make more sense

No, not in this case wit hthe french-quebec pops.

File: common/national_focus/canada.txt (hunk 1 above)
Comment (AnonymousRevan) [reply common/national_focus/canada.txt:2508]: > Why are we buffing France in the Canada tree?

see convo in discord

**Review by GrenadierSchubes — APPROVED** [review]
Did what he said he would, as well as answered the questions

**Review by Mookubooku — APPROVED** [review]
Looks good.

**Pull request merged**